### PR TITLE
Showcase que jamear.com

### PR DESCRIPTION
**Pull request by devrchancay**
adding a new website to showcase and updating URL for another project.

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -6796,8 +6796,8 @@
   description: >-
     Recipes and videos with the best of Ecuadorian cuisine.
     Collectable recipes from Diario El Universo.
-  main_url: https://recetas-eu.netlify.com/
-  url: https://recetas-eu.netlify.com/
+  main_url: https://recetas-eu.now.sh/
+  url: https://recetas-eu.now.sh/
```
Comment (blainekasten) [inline docs/sites.yml:6800]: I don't understand why you changed this? Are you involved with this project? Both links seem to work. Seems out of scope of you adding the Quejamear showcase

**Comment by blainekasten** [comment]
I'm happy to merge this. Could we just revert the changes to the other showcase not to not bloat scope on this?

**Comment by devrchancay** [comment]
I wanted to take advantage of the fact that I migrated my previous site to Vercel.

@blainekasten  is it a lot of trouble to merge both in this PR?

File: docs/sites.yml (hunk 1 above)
Comment (devrchancay) [reply docs/sites.yml:6800]: I understand that it is out of scope, but I wanted to take advantage of the PR, to update one of my previous sites.

**Comment by devrchancay** [comment]
@blainekasten can you help me with this PR?
